src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -65,42 +78,56 @@ const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({
         }
     };
 
+    const toggleCollapse = () => setCollapsed(!collapsed);
+
     return (
         <Box className={ classes.root } display="flex" height="100vh">
             <OrganizeSidebar />
             <Box display="flex" flexDirection="column" height="100vh" overflow="auto" position="relative" width={ 1 }>
                 <Box display={ fixedHeight ? 'flex' :'block' } flexDirection="column" height={ fixedHeight ? 1 : 'auto' }>
                     <Box component="header" flexGrow={ 0 } flexShrink={ 0 }>
-                        <Box mb={ 2 } pt={ 3 } px={ 3 }>
-                            <BreadcrumbTrail/>
-                            { /* Title, subtitle, and action buttons */ }
-                            <Box className={ classes.titleGrid } mt={ 2 }>
-                                <Box overflow="hidden">
-                                    <Typography className={ classes.title } component="h1" data-testid="page-title" noWrap variant="h3">
-                                        { title }
-                                    </Typography>
-                                    <Typography component="h2" variant="h5">
-                                        { subtitle }
-                                    </Typography>
-                                </Box>
-                                <Box>
-                                    { actionButtons }
+                        <Box mb={ collapsed ? 0 : 2 } pt={ 3 } px={ 3 }>
+                            <Box display="flex" flexDirection="row" justifyContent="space-between">
+                                <BreadcrumbTrail highlight={ collapsed } />
+                                <Box className="hide-button">
+                                    <Button onClick={ toggleCollapse } size="small"
+                                        startIcon={ collapsed ? <ArrowDownward /> : <ArrowUpward /> }>
+                                        { collapsed ? 'expand' : 'collapse' } header
```

Comment:
This should be localised! You can do this by adding an english localised text in a `.yml` file in `src/locale/` you can put it wherever it makes sense. Maybe `misc/en.yml`. You can then implement this with `<FormattedMessage id={...}/>`

Then, for testing, our test runner doesn't localise text but shows the id used to look up text. So you could assert that the id of the text you want is rendered.

Reply:
Can't believe I forgot about this, since I had actually spent some time reflecting on how great it is that this project is already internationalised (at least in 2 languages at this point anyway...)
Fixed in latest commit :)